products/mlengine/api.yaml

```diff
@@ -235,4 +235,338 @@ objects:
+                    - ACCELERATOR_TYPE_UNSPECIFIED
+                    - NVIDIA_TESLA_K80	
+                    - NVIDIA_TESLA_P100
+                    - NVIDIA_TESLA_V100
+                    - NVIDIA_TESLA_P4
+                    - NVIDIA_TESLA_T4
+                    - TPU_V2
+                    - TPU_V3 
+              - !ruby/object:Api::Type::String
+                name: 'imageUri'
+                description: The Docker image to run on the replica. This image must be in Container Registry.
+              - !ruby/object:Api::Type::String
+                name: 'tpuTfVersion'
+                description: |
+                  The AI Platform runtime version that includes a TensorFlow version matching the one used in the 
+                  custom container. This field is required if the replica is a TPU worker that uses a custom container. 
+                  Otherwise, do not specify this field.
+          - !ruby/object:Api::Type::String
+            name: 'workerType'
+            description: Specifies the type of virtual machine to use for your training job's worker nodes.
+            required: false
+          - !ruby/object:Api::Type::NestedObject
+            name: 'workerConfig'
+            description: The configuration for workers.
+            required: false
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'acceleratorConfig'
+                description: Represents the type and number of accelerators used by the replica
+                properties:
+                - !ruby/object:Api::Type::Integer
+                  name: 'count'
+                  description: The number of accelerators to attach to each machine running the job.
+                - !ruby/object:Api::Type::Enum
+                  name: 'type'
+                  description: The type of accelerator to use.
+                  values:
+                    - ACCELERATOR_TYPE_UNSPECIFIED
+                    - NVIDIA_TESLA_K80	
+                    - NVIDIA_TESLA_P100
+                    - NVIDIA_TESLA_V100
+                    - NVIDIA_TESLA_P4
+                    - NVIDIA_TESLA_T4
+                    - TPU_V2
+                    - TPU_V3 
+              - !ruby/object:Api::Type::String
+                name: 'imageUri'
+                description: The Docker image to run on the replica. This image must be in Container Registry.
+              - !ruby/object:Api::Type::String
+                name: 'tpuTfVersion'
+                description: |
+                  The AI Platform runtime version that includes a TensorFlow version matching the one used in the 
+                  custom container. This field is required if the replica is a TPU worker that uses a custom container. 
+                  Otherwise, do not specify this field.
+          - !ruby/object:Api::Type::String
+            name: 'parameterServerType'
+            description: The configuration for parameter servers
```

Comment:
This looks like the description for `parameterServerConfig` instead

Reply:
corrected